x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -494,17 +495,24 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
       : i18n.CREATE_RULE_EXCEPTION;
   }, [listType]);
 
+  const exceptionFlyoutTitleId = useGeneratedHtmlId({
+    prefix: 'exceptionFlyoutTitle',
```

Comment:
should it be `addExceptionFlyoutTitle`?

Reply:
@vitaliidm it's not so important, I just copied that from `data-test-subj `for consistency

<img width="567" alt="image" src="https://github.com/elastic/kibana/assets/20072247/9d7f83f6-ef44-4557-9415-bee28d6a9da1">

Reply:
fair enough.
I noticed some of other elements have `data-test-subj="addExceptionFlyout"` and so on, with little more details.